ansible/vars.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 runtime_user: "indicators"
 jenkins_user: "jenkins"
-jenkins_artifact_dir: "/var/lib/jenkins/artifacts"
+#jenkins_artifact_dir: "/var/lib/jenkins/artifacts" # TODO: Switcheroo to `artifacts` after this goes live! 
```

Comment:
Should the switch to `artifacts` be done in this PR, or in a separate PR after this one is merged?

Reply:
I think we should do it in this PR. I have the other directories configured mostly just in case this were to be merged early so as not to clobber anything.